internal/commands/apply_test.go

```diff
@@ -155,7 +155,7 @@ func TestApplyNegative(t *testing.T) {
 			asserter: func(s *scaffold, err error) {
 				a := assert.New(s.t)
 				a.True(cmd.IsUsageError(err))
-				a.Equal("exactly one environment required", err.Error())
+				a.Equal("exactly one environment required, but provided: [dev prod]", err.Error())
```

Comment:
Can we add a case for empty string "" args?

Reply:
Sure, but that will provide different error. which is `invalid environment`